_shows/90_91/frosted_icing.md

```diff
@@ -5,8 +5,8 @@ season: In House
 season_sort: 20
 period:
```

Comment:
Can we make the assumption that this is `period: Autumn` and `venue: New Theatre`?

Reply:
Probably yeah